cp2/ControlPanel/ControlPanel/view.py

```diff
@@ -726,12 +677,14 @@ def ajax_add_demo_extras(request):
 def ajax_delete_demo_extras(request):
     demo_id = request.GET["demo_id"]
     if user_can_edit_demo(request.user, demo_id):
-        response, _ = api_post(
-            "/api/demoinfo/delete_demoextras",
-            method="post",
-            params={"demo_id": demo_id},
+        _, status_code = api_post(
```

Comment:
If I understand correctly the "remove file" button in the CP2 actually removes all demoextras of a demo?

Reply:
It removes it from "staticData" folder. Just in case, there can only be one demoextras per demo